projects/plugins/boost/readme.txt

```diff
@@ -165,52 +165,71 @@ If you run into compatibility issues, please do let us know. You can drop us a l
 1. Manage your Jetpack Boost settings
 
 == Changelog ==
+### 1.2.0-beta - 2021-08-10
+#### Added
+- Critical CSS: Added a new Advanced Critical CSS recommendations page.
+
+#### Changed
+- Critical CSS: Updated error reporting for Critical CSS to offer more users more guidance.
+- Updated package dependencies.
+- Tooling: Moved all development to the Jetpack monorepo.
+
+#### Fixed
+- Tooling: Fix PHP unit testing dependency on later versions of PHP.
+
+## 1.1.0 - 2021-06-17
+
+- Update: User connection is no longer required for Speed Scores.
+- Update: Completely revamped how site speed scores are retreived.
+- Update: Reduced backend dashboard JavaScript bundle size.
+- Update: Added a message to explain how site score is calculated.
+- Update: Added "Offline Mode" to allow testing Jetpack Boost on local environments easily.
+- Update: Improved error handling and the error messages provided.
+- Update: Improved Critical CSS Generation stability.
+- Update: Remove animations from Critical CSS.
+- Fix: Incompatibility with UsersWP and similar plugins that might introduce redirects during Critical CSS Generation.
+
+## 1.0.6 - 2021-05-25
+
+- Fix: Failed to execute 'json' errors
+- Fix: Connection UI Border issues
+- Update: Improve Jetpack compatibility
+- Update: Improve Critical CSS Compatibility with caching and minification plugins
+- Update: Clean up JavaScript dependencies
+
+## 1.0.5 - 2021-05-13
+
+- Fixed: Defer JavaScript compatibility with XML Requests
+
+## 1.0.4 - 2021-05-06
+
+- Fixed: Web Stories compatibility
+- Improved: "Defer Non-Essential Javascript" module compatibility with other plugins
+
+## 1.0.3 - 2021-04-26
+
+- Updated: Support for AMP Plugin 2.0+
+- Updated: No longer defer JavaScript on POST, AJAX, Cron requests and sitemaps.
+
+## 1.0.2 - 2021-04-22
+
+- Improved: HTML Media tag handling
+- Fixed: Metrics timeout caused by caching in the REST API
+
+## 1.0.1 - 2021-04-20
+
+- Fixed: An issue where the connection iframe would sometimes break
+- Updated: On connection: showing an XML RPC Error instead of HTTP 500 when XML-RPC is disabled
+
+## 1.0.0 - 2021-04-19
```

Comment:
I'd suggest going one level down with the headings, for a better display on the WordPress.org page:

```suggestion
### 1.1.0 - 2021-06-17

- Update: User connection is no longer required for Speed Scores.
- Update: Completely revamped how site speed scores are retreived.
- Update: Reduced backend dashboard JavaScript bundle size.
- Update: Added a message to explain how site score is calculated.
- Update: Added "Offline Mode" to allow testing Jetpack Boost on local environments easily.
- Update: Improved error handling and the error messages provided.
- Update: Improved Critical CSS Generation stability.
- Update: Remove animations from Critical CSS.
- Fix: Incompatibility with UsersWP and similar plugins that might introduce redirects during Critical CSS Generation.

### 1.0.6 - 2021-05-25

- Fix: Failed to execute 'json' errors
- Fix: Connection UI Border issues
- Update: Improve Jetpack compatibility
- Update: Improve Critical CSS Compatibility with caching and minification plugins
- Update: Clean up JavaScript dependencies

### 1.0.5 - 2021-05-13

- Fixed: Defer JavaScript compatibility with XML Requests

### 1.0.4 - 2021-05-06

- Fixed: Web Stories compatibility
- Improved: "Defer Non-Essential Javascript" module compatibility with other plugins

### 1.0.3 - 2021-04-26

- Updated: Support for AMP Plugin 2.0+
- Updated: No longer defer JavaScript on POST, AJAX, Cron requests and sitemaps.

### 1.0.2 - 2021-04-22

- Improved: HTML Media tag handling
- Fixed: Metrics timeout caused by caching in the REST API

### 1.0.1 - 2021-04-20

- Fixed: An issue where the connection iframe would sometimes break
- Updated: On connection: showing an XML RPC Error instead of HTTP 500 when XML-RPC is disabled

### 1.0.0 - 2021-04-19
```